_src/_guide/_setup.md

```diff
@@ -21,30 +21,22 @@ pip install -U bigchaindb-driver
 </dependency>
 ```
 
-Then, include that as a module and connect to any BigchainDB node. You can create your own `app_id` and `app_key` on [BigchainDB Testnet](https://testnet.bigchaindb.com).
+Then, include that as a module and connect to any BigchainDB node.
 
 ```js
 const BigchainDB = require('bigchaindb-driver')
 
 const API_PATH = 'https://test.bigchaindb.com/api/v1/'
```

Comment:
at least in the markdown & html files we can source the api url from the `bigchaindb_api_url` value in `_config.yml`. So here we can simply use liquid variable: `{{ site.bigchaindb_api_url }}/api/v1/`